packages/block-library/README.md

```diff
@@ -119,4 +119,23 @@ To find out more about contributing to this package or Gutenberg as a whole, ple
     }
     ```
 
+### Naming convention for PHP functions
+
+All PHP function names within the subdirectories of the `packages/block-library/src/` directory should start with one of the following prefixes:
```

Comment:
Should it be clarified that it applies to function definitions? Example:
```suggestion
All declared PHP function names within the subdirectories of the `packages/block-library/src/` directory should start with one of the following prefixes:
```

Reply:
Good catch. Fixed in 0586c6351a3cc255078ee5aea63915b644a87c42